sdk/metricsadvisor/ai-metrics-advisor/samples/typescript/src/quickstart.ts

```diff
@@ -0,0 +1,298 @@
+  const credential = new MetricsAdvisorKeyCredential(subscriptionKey, apiKey);
+
+  const client = new MetricsAdvisorClient(endpoint, credential);
+  const adminClient = new MetricsAdvisorAdministrationClient(endpoint, credential);
+
+  const created = await createDataFeed(adminClient);
+  console.log(`Data feed created: ${created.id}`);
+  console.log("  metric ids: ");
+  console.log(created.metricIds);
+
+  console.log("Waiting for a minute before checking ingestion status...");
+  await delay(60 * 1000);
+
+  try {
+    await checkIngestionStatus(
+      adminClient,
+      created.id,
+      new Date(Date.UTC(2020, 8, 1)),
+      new Date(Date.UTC(2020, 8, 12))
+    );
+
+    const metricId = created.metricIds[0];
+    const detectionConfig = await configureAnomalyDetectionConfiguration(adminClient, metricId);
+    console.log(`Detection configuration created: ${detectionConfig.id!}`);
+
+    const hook = await createWebhookHook(adminClient);
+    console.log(`Webhook hook created: ${hook.id!}`);
+
+    const alertConfig = await configureAlertConfiguration(adminClient, detectionConfig.id!, [
+      hook.id!
+    ]);
+    console.log(`Alert configuration created: ${alertConfig.id!}`);
+
+    // you can use alert configuration created in above step to query the alert.
+    const alertIds = await queryAlerts(
+      client,
+      alertConfig.id!,
+      new Date(Date.UTC(2020, 8, 1)),
+      new Date(Date.UTC(2020, 8, 12))
+    );
+
+    if (alertIds.length > 1) {
+      // query anomalies using an alert id.
+      await queryAnomaliesByAlert(client, alertConfig.id!, alertIds[0]);
+    } else {
+      console.log("No alerts during the time period");
+    }
+  } finally {
+    console.log(`Deleting the data feed '${created.id!}`);
+    await adminClient.deleteDataFeed(created.id!);
+  }
+}
+
+async function createDataFeed(
+  adminClient: MetricsAdvisorAdministrationClient
+): Promise<GetDataFeedResponse> {
+  const metric: Metric[] = [
+    {
+      name: "revenue",
+      displayName: "revenue",
+      description: "Metric1 description"
+    },
+    {
+      name: "cost",
+      displayName: "cost",
+      description: "Metric2 description"
+    }
+  ];
+  const dimension: Dimension[] = [
+    { name: "city", displayName: "city display" },
+    { name: "category", displayName: "category display" }
+  ];
+  const dataFeedSchema: DataFeedSchema = {
+    metrics: metric,
+    dimensions: dimension,
+    timestampColumn: undefined
+  };
+  const dataFeedIngestion: DataFeedIngestionSettings = {
+    ingestionStartTime: new Date(Date.UTC(2020, 5, 1)),
+    ingestionStartOffsetInSeconds: 0,
+    dataSourceRequestConcurrency: -1,
+    ingestionRetryDelayInSeconds: -1,
+    stopRetryAfterInSeconds: -1
+  };
+  const granualarity: DataFeedGranularity = {
+    granularityType: "Daily"
+  };
+  const source: DataFeedSource = {
+    dataSourceType: "SqlServer",
+    dataSourceParameter: {
+      connectionString:
```

Comment:
This connection string is public and published at doc site: https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor/quickstarts/rest-api#add-a-data-feed-from-a-sample-or-data-source